src/AutoMapper/ConfigurationStore.cs

```diff
@@ -364,6 +364,17 @@ public TypeMap FindTypeMapFor(TypePair typePair)
             return typeMap;
         }
 
+        private bool CoveredByObjectMap(TypePair typePair)
+        {
+            return GetMappers().FirstOrDefault(m => m.IsMatch(new ResolutionContext(null, null, typePair.SourceType, typePair.DestinationType, new MappingOperationOptions(), new MappingEngine(this)))) != null;
+        }

```

Comment:
I'll just merge this and fix it up with getting rid of the ResolutionContext stuff. Be annoying if I pushed a changed and screwed you over at this point.